test/p2p/sanity.spec.ts

```diff
@@ -52,8 +53,8 @@ describe('P2P Sanity Tests', () => {
   let unusedPort: number;
 
   before(async () => {
-    nodeOneConfig = createConfig(1, 0);
-    nodeTwoConfig = createConfig(2, 0);
+    nodeOneConfig = createConfig(1, 8855);
```

Comment:
I think we want to keep these ports dynamic for testing purposes, any particular reason this was changed?